pkgs/tools/cd-dvd/srt-to-vtt-cl/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, which, substituteAll }:
+
+stdenv.mkDerivation rec {
+  pname = "srt-to-vtt-cl";
```

```suggestion
  pname = "srt-to-vtt-cl-unstable";
```

In case we are using the commit not a stable release.